easybuild/easyconfigs/p/PyTorch/PyTorch-1.4.0-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -242,17 +241,18 @@ checksums = [
     ('5cb33c5d51047c96c8efa57e2d3dde7723e3b358eb1bbc97d793f4e26fd17481',
      'de7e4df12f4c474000ecc83dadc386f16337b5f9ecf13c1c5eae6314bf444c6f'),  # mkl-dnn-20190928.tar.gz
     '001c9bf604aebe4b39ccad15332a71130b07b780c539ceca84d6c64cd6fc8a68',  # PyTorch-1.2.0_fix-findAVX.patch
-    'c4183bcb29a8bcbadea0341e93a3a32afdf860aa31331b768e787d899183da92',  # PyTorch-1.2.0_disable-tests-ppc64le.patch
     '57cc8d2159a35de89139729c5390eb944332b9907d97b2b85a0b6ded30bd198e',  # PyTorch-1.4.0_fix-missing-sleef_h.patch
+    '8782d68d4265cbb89a7908124b0a9a469fd89ee46c7c78a02a5c623ece457d6e',  # PyTorch-1.4.0-add-cuda11-support.patch
+    '91977512b45aa2957b715a3e8d2b04016448b60407423de69d604fdc743d5f69',  # PyTorch-1.4.0-fix-test-prec.patch
 ]
 
 builddependencies = [
     ('CMake', '3.15.3'),
     ('hypothesis', '4.44.2', versionsuffix),
-    ('Ninja', '1.9.0'),
 ]
 
 dependencies = [
+    ('Ninja', '1.9.0'),
```

Comment:
Can you clarify why `Ninja` was made a runtime dep?

Reply:
Because PyTorch can JIT compile C++ extension and uses Ninja for that. This is what PyTorch-check-cpp-extension.py  is testing

Reply:
OK, let's clarify that with a comment above this line, because it looks like a mistake at first sight.